cmd/create/accountroles/cmd.go

```diff
@@ -182,12 +182,30 @@ func run(cmd *cobra.Command, argv []string) {
 			"any supported ROSA version can be installed with managed policies")
 	}
 
+	r.WithAWS()
+
+	// Check whether or not AWS account is govcloud
+	callerIdentityOutput, err := r.AWSClient.GetCallerIdentity()
+	if err != nil {
+		r.Reporter.Warnf("Unable to get AWS Caller Identity.")
+		os.Exit(1)
+	}
+
+	callerIsGovcloud := false
+
+	if (*callerIdentityOutput.Arn)[:14] == "arn:aws-us-gov" {
```

Comment:
What is the ARN is nil, this will crash, best to use `aws.StringValue(callerIdentityOutput.Arn)` to safely deref the string. 

aws have an arn Parse function, we can then pass the string value too, and from there we can check the `Partition` value.

Reply:
not sure i understand the Parse/Partition comment, but i will implement the first

Reply:
now understood